pombola/core/views.py

```diff
@@ -611,6 +611,31 @@ def handle_approx_date(date):
 
         return response
 
+    elif request.GET.get('format') == 'popolo':
+
+        persons = []
+
+        for position in positions:
```

Comment:
This API seems quite slow at the moment. Perhaps we could filter down the list of positions and fetch the related people to speed things up?

```python
positions.filter(person__contacts__kind__slug='email').select_related('person')
```